### PR TITLE
Fix plans for Repeat(trail) and NullifyMetaData

### PR DESCRIPTION
**Pull request by JPryce-Aklundh**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-1043.surge.sh

Updated pages:
- [Operators in detail](https://neo4j-docs-cypher-1043.surge.sh/cypher-manual/5/planning-and-tuning/operators/operators-detail)

File: modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc [hunk 1]
```diff
@@ -3970,6 +3970,7 @@ Total database accesses: 49, total allocated memory: 1200
 // Repeat(Trail)
 
 Given a start node, the `Repeat(Trail)` operator will traverse xref::patterns/variable-length-patterns.adoc#quantified-path-patterns[quantified path patterns] that cannot be solved with the xref::planning-and-tuning/operators/operators-detail.adoc#query-plan-varlength-expand-all[`VarLengthExpand(All)`] operator.
+Specifically, `Repeat(Trail)` is used to solve quantified path patterns including at least two relationships, or a pre-filter.
```
Comment (WilcoNeo) [inline modules/ROOT/pages/planning-and-tuning/operators/operators-detail.adoc]: Is the concept of a `pre-filter` clear? It is explained somewhere in the docs?

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Review by WilcoNeo — APPROVED** [review]
Thanks for updating this!

**Pull request merged**